apis/resourcecontrollerv2/v1alpha1/resourceinstance_types.go

```diff
@@ -111,6 +108,9 @@ type ResourceInstanceObservation struct {
 	// The sub-type of instance, e.g. `cfaas`.
 	SubType string `json:"subType,omitempty"`
 
+	// A boolean that dictates if the resource instance is locked or not.
+	Locked bool `json:"locked,omitempty"`
+
```

Comment:
Not sure if you left this one out on purpose or just overlooked it.  If we need to exclude it that is pretty easy to do -- we would just add an `x-crossplane-exclude; true` annotation on this property -- the generator already understands that.

Reply:
I think I overlooked, or thought it was the same as `EntityLock` which is in the Request